### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by richiehx**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -117,28 +117,25 @@ The `UI` component,
...
-from storage component to save the data to local storage each time the data is modified.
+It interacts with the `UI` component by taking the user input from it, and setting the UI display accordingly.
+It also calls APIs (addPerson, deleteLesson for example) from the `Model` component to modify the data representation, and call APIs
+from the `Storage` component to save the data to local storage each time the data is modified.
 
 The sequence diagram below illustrates the interactions within the `Logic` component, taking `execute("delete 1")` API call as an example.
 <puml src="diagrams/DeleteSequenceDiagram.puml" alt="Interactions Inside the Logic Component for the `delete 1` Command" />
 
-Inside the `Logic` component, there are 4 main components: LogicManager, AddressBookParser, ScheduleParser and Command.
+Inside the `Logic` component, there are 4 main components: `LogicManager`, `AddressBookParser`, `ScheduleParser` and `Command`.
```
Comment (elaineshijie) [inline docs/DeveloperGuide.md]: Hmmm I think we don't have a ScheduleParser?

Comment (richiehx) [reply docs/DeveloperGuide.md]: Hmm, that seems to be the case. I just added the `` brackets but i'll remove the ScheduleParser reference.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -117,28 +117,25 @@ The `UI` component,
...
-from storage component to save the data to local storage each time the data is modified.
+It interacts with the `UI` component by taking the user input from it, and setting the UI display accordingly.
+It also calls APIs (addPerson, deleteLesson for example) from the `Model` component to modify the data representation, and call APIs
+from the `Storage` component to save the data to local storage each time the data is modified.
 
 The sequence diagram below illustrates the interactions within the `Logic` component, taking `execute("delete 1")` API call as an example.
 <puml src="diagrams/DeleteSequenceDiagram.puml" alt="Interactions Inside the Logic Component for the `delete 1` Command" />
 
-Inside the `Logic` component, there are 4 main components: LogicManager, AddressBookParser, ScheduleParser and Command.
+Inside the `Logic` component, there are 3 main components: `LogicManager`, `AddressBookParser` and `Command`.
```
Comment (yiwen101) [inline docs/DeveloperGuide.md:130]: A bit weird writing "AddressBookParser" here, how about says "there are 4 main components ...", as addressBookParser does not implement parser.or could simply says "Parser".

Comment (yiwen101) [reply docs/DeveloperGuide.md:130]: or says there are 4 main components, as addressBookParser does not implement parser

**Review by yiwen101 — APPROVED** [review]
LGTM, just one minor suggestion of change

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (richiehx) [reply docs/DeveloperGuide.md:130]: For this section, I interpret it as talking about the various important classes of the Logic component so like the 3 important classes LogicManager, AddressBookParser and the Command classes. Not too sure how we want to phrase it.

Comment (yiwen101) [reply docs/DeveloperGuide.md:130]: sure

**Pull request merged**